packages/next-server/lib/head.js

```diff
@@ -2,10 +2,10 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import sideEffect from './side-effect'
 
+export const HeadManagerContext = React.createContext()
```

Comment:
I think this should be moved to a separate file so that we're 100% sure it will only bundle this line instead of always bundling the `<Head>` `next/head` component even when it's not being used.